lib/clustering/cluster_ht_item.rb

```diff
@@ -22,79 +20,16 @@ def initialize(*ht_items)
       end
     end
 
-    def cluster(getter: cluster_getter)
-      # For the case where there are no OCNs, ClusterGetter always creates a new
-      # cluster. We might want to first consider searching by item ID when there
-      # are no OCNs.
+    def cluster
+      return if ocns.empty?
 
-      getter.get do |cluster|
-        update_or_add_ht_items(cluster)
-      end
-    end
-
-    # Removes an HTItem
-    def delete
```

Comment:
Removed this all for now since we aren't persisting anything about ht items other than OCNs in the same cluster (as the comment in the new stuff indicates)